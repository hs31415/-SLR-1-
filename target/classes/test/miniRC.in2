integer a[5][10];