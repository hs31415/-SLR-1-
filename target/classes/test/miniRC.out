14integer
id:a
38[
int:5
38[
int:10
44;
